charts/app-config/values-integration.yaml

```diff
@@ -1053,7 +1053,7 @@ govukApplications:
         <<: *alb-ingress-www-waf-ruleset
         alb.ingress.kubernetes.io/load-balancer-name: assets-origin
         alb.ingress.kubernetes.io/group.name: assets-origin
-        alb.ingress.kubernetes.io/group.order: '15'
+        alb.ingress.kubernetes.io/group.order: '16'
         alb.ingress.kubernetes.io/security-groups: eks_ingress_www_origin
```

Comment:
Is this going to cause problems? It was removing a similar line yesterday that fixed the draft assets problem.

Reply:
Hmm, I _think_ we do want integration to be access limited.

Reply:
Ah, it's just integration - yes, sorry ignore me!

Reply:
I dunno — staging definitely is supposed to be GDS-only but don't forget we have this whacky use of integration for training content people in other departments.